### PR TITLE
add vpc binding api

### PR DESCRIPTION
**Pull request by jhstianwei**

### CODE REVIEW

File: src/test/java/com/aliyuncs/fc/FunctionComputeClientTest.java [hunk 1]
```diff
@@ -2234,6 +2234,41 @@ private FunctionMetadata getFunction(String functionName, FunctionMetadata[] fun
...
+        CreateServiceRequest createSReq = new CreateServiceRequest();
+        createSReq.setServiceName(serviceName);
+        createSReq.setRole(ROLE);
+        CreateServiceResponse response = client.createService(createSReq);
+        assertEquals(serviceName, response.getServiceName());
+        assertFalse(Strings.isNullOrEmpty(response.getRequestId()));
+        assertFalse(Strings.isNullOrEmpty(response.getServiceId()));
+        assertEquals(ROLE, response.getRole());
+
+        // create vpc bind
```
Comment (rockuw) [inline src/test/java/com/aliyuncs/fc/FunctionComputeClientTest.java:2252]: vpc binding

File: src/test/java/com/aliyuncs/fc/FunctionComputeClientTest.java [hunk 2]
```diff
@@ -2234,6 +2234,41 @@ private FunctionMetadata getFunction(String functionName, FunctionMetadata[] fun
...
+        CreateVpcBindingRequest request = new CreateVpcBindingRequest();
+        request.setServiceName(serviceName).setVpcId(VPC_ID);
+        CreateVpcBindingResponse createVpcBindingResponse = client.createVpcBinding(request);
+        assertEquals(200, createVpcBindingResponse.getStatus());
+
+        // list vpc bind
```
Comment (rockuw) [inline src/test/java/com/aliyuncs/fc/FunctionComputeClientTest.java:2258]: vpc binding

File: src/test/java/com/aliyuncs/fc/FunctionComputeClientTest.java [hunk 3]
```diff
@@ -2234,6 +2234,41 @@ private FunctionMetadata getFunction(String functionName, FunctionMetadata[] fun
...
+        ListVpcBindingsRequest listVpcBindingsRequest = new ListVpcBindingsRequest();
+        listVpcBindingsRequest.setServiceName(serviceName);
+        ListVpcBindingsResponse listVpcBindingsResponse = client.listVpcBindings(listVpcBindingsRequest);
+        assertEquals(1, listVpcBindingsResponse.getVpcIDs().length);
+        assertEquals(VPC_ID, listVpcBindingsResponse.getVpcIDs()[0]);
+
+        // delete vpc bind
```
Comment (rockuw) [inline src/test/java/com/aliyuncs/fc/FunctionComputeClientTest.java:2265]: vpc binding

File: src/test/java/com/aliyuncs/fc/FunctionComputeClientTest.java (hunk 1 above)
Comment (jhstianwei) [reply src/test/java/com/aliyuncs/fc/FunctionComputeClientTest.java:2252]: done

File: src/test/java/com/aliyuncs/fc/FunctionComputeClientTest.java (hunk 2 above)
Comment (jhstianwei) [reply src/test/java/com/aliyuncs/fc/FunctionComputeClientTest.java:2258]: done

Comment (jhstianwei) [reply src/test/java/com/aliyuncs/fc/FunctionComputeClientTest.java:2258]: done

File: src/test/java/com/aliyuncs/fc/FunctionComputeClientTest.java (hunk 3 above)
Comment (jhstianwei) [reply src/test/java/com/aliyuncs/fc/FunctionComputeClientTest.java:2265]: done

**Pull request merged**